asab/library/service.py

```
@@ -382,6 +387,94 @@ async def _list(self, path, providers):
+			- self.FavoritePaths: [ ('/path/folder/', ['tenant', '*', ...]), ... ]
+		Expected YAML shape:
+			/path:
+				tenants:
+				- system
+		"""
+		old_favorites = self.Favorites.copy()
+		old_favorite_paths = list(self.FavoritePaths)
+
+		try:
+			fav_file = await self.Libraries[0].read('/.favorites.yaml')
+		except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)